src/x509.c

```diff
@@ -7417,7 +7416,7 @@ WOLFSSL_API int wolfSSL_X509_load_crl_file(WOLFSSL_X509_LOOKUP *ctx,
         }   while(crl == NULL);
 
         ret = count;
-    } else if (type == WOLFSSL_FILETYPE_ASN1) {
+    } else if (type == X509_FILETYPE_ASN1) {
```

Comment:
Add newline before `else if`. Use `WOLFSSL_FILETYPE_ASN1` not our compatibility macro.